### DISCUSSION TITLE
Adding audio unit support for Logic

### DISCUSSION THREAD

**Discussion by alittlesound** [DISTRHO/Cardinal#157] [Ideas]
Just curious if there was a way to easily port this over to audio unit format or not.  Thanks!

**Comment by falkTX** [comment]
Easy depends on how far you are willing to go.
I started to make a plugin wrapper around Cardinal using juce, to allow the juce formats exposing Cardinal stuff.
If you are comfortable developing with juce, I can give some pointers on where to start.
